test/cmdlineTests/exp_base_literal/output

```diff
@@ -208,78 +208,157 @@ object "C_81" {
+                /// @src 0:204,219
                 let var_x_28 := expr_33
+                /// @src 0:232,237
+                /// @src 0:232,234
                 let expr_37 := 0x0a
+                /// @src 0:236,237
+                /// @src 0:236,237
                 let _9 := var_c_8
                 let expr_38 := _9
+                /// @src 0:232,237
                 let _10 := convert_t_rational_10_by_1_to_t_uint256(expr_37)
                 let expr_39 := checked_exp_t_rational_10_by_1_t_uint256(expr_38)
+                /// @src 0:223,237
                 let var_y_36 := expr_39
+                /// @src 0:250,265
+                /// @src 0:250,262
+                /// @src 0:251,260
                 let expr_47 := 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff
+                /// @src 0:250,262
                 let expr_48 := expr_47
+                /// @src 0:264,265
+                /// @src 0:264,265
                 let _11 := var_d_10
                 let expr_49 := _11
+                /// @src 0:250,265
                 let _12 := convert_t_rational_115792089237316195423570985008687907853269984665640564039457584007913129639935_by_1_to_t_uint256(expr_48)
                 let expr_50 := checked_exp_t_rational_115792089237316195423570985008687907853269984665640564039457584007913129639935_by_1_t_uint256(expr_49)
+                /// @src 0:241,265
                 let var_z_42 := expr_50
+                /// @src 0:307,313
+                /// @src 0:307,310
+                /// @src 0:308,309
                 let expr_53 := 0x00
+                /// @src 0:307,310
                 let expr_54 := expr_53
+                /// @src 0:312,313
+                /// @src 0:312,313
                 let _13 := var_a_4
                 let expr_55 := _13
+                /// @src 0:307,313
                 let _14 := convert_t_rational_0_by_1_to_t_uint256(expr_54)
                 let expr_56 := checked_exp_t_rational_0_by_1_t_uint256(expr_55)
+                /// @src 0:303,313
+                /// @src 0:303,304
+                /// @src 0:303,304
+                /// @src 0:303,313
                 var_w_22 := expr_56
                 let expr_57 := expr_56
+                /// @src 0:321,328
+                /// @src 0:321,325
+                /// @src 0:323,324
                 let expr_60 := 0x01
+                /// @src 0:322,324
                 let expr_61 := 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff
+                /// @src 0:321,325
                 let expr_62 := expr_61
+                /// @src 0:327,328
+                /// @src 0:327,328
                 let _15 := var_b_6
                 let expr_63 := _15
+                /// @src 0:321,328
                 let _16 := convert_t_rational_minus_1_by_1_to_t_int256(expr_62)
                 let expr_64 := checked_exp_t_rational_minus_1_by_1_t_uint256(expr_63)
+                /// @src 0:317,328
```

Comment:
Can we somehow avoid mentioning source location multiple times without any new code being emitted?

Reply:
Maybe we should change the use of `m_code << ` to something like ``emitCode() << `` where `emitCode` is a function that prepends the current source location if it has changed since the previous call.